front/app/containers/ProjectsShowPage/shared/header/ProjectHeader.tsx

```diff
@@ -37,6 +38,15 @@ const Container = styled.div`
   `}
 `;
 
+const TopBar = styled.div`
+  display: flex;
+  justify-content: space-between;
+
+  ${isRtl`
+    flex-direction: row-reverse;
+  `}
```

Comment:
FYI: we're not actively maintaining/supporting rtl at the moment, so I wouldn't worry too much about adding it to new styles.

Reply:
Ok, I just copy/pasted this code from the `TopBar` in the [IdeasShow](https://github.com/CitizenLabDotCo/citizenlab/blob/master/front/app/containers/IdeasShow/index.tsx#L157)... should I just leave it, or remove it in both places?

Reply:
I think it’s fine to leave it as it is in both places for now.